Tools/Scripts/build-webkit

```diff
@@ -224,10 +222,14 @@ if (!isCMakeBuild()) {
     }
 }
 
-if ((isAppleWebKit() || isWin() || isPlayStation()) && !-d "WebKitLibraries") {
+if ((isAppleWebKit() || isPlayStation()) && !-d "WebKitLibraries") {
     die "Error: No WebKitLibraries directory found. Please do a fresh checkout.\n";
 }
 
+if (isAnyWindows() && ! defined $ENV{"VCPKG_ROOT"}) {
```

Comment:
isAnyWindows seems to be "is running on windows or cygwin as it checks $^O, not doing the windows port (which I think is isWin).  This seems wrong since I think we only want to be using vcpkg for the one port.